addons/mail/static/tests/chat_window/chat_window.test.js

```diff
@@ -1076,3 +1076,20 @@ test("open channel in chat window from push notification", async () => {
     );
     await contains(".o-mail-ChatWindow", { text: "General" });
 });
+
+test("Ctrl + K opens command palette in conversation mode in chat window", async () => {
+    const pyEnv = await startServer();
+    pyEnv["discuss.channel"].create([
+        {
+            name: "General",
+            channel_member_ids: [
+                Command.create({ partner_id: serverState.partnerId, fold_state: "open" }),
+            ],
+        },
+    ]);
+    await start();
+    await contains(".o-mail-ChatWindow", { text: "General" });
+    await focus(".o-mail-ChatWindow");
```

Comment:
Both lines could be grouped, as we know the chat window exists if we can focus it.